src/Console/Relay.php

```diff
@@ -0,0 +1,150 @@
+<?php
+/**
+ * @copyright Copyright (C) 2020, Friendica
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace Friendica\Console;
+
+use Asika\SimpleConsole\CommandArgsException;
+use Friendica\App;
+use Friendica\Database\DBA;
+use Friendica\Model\APContact;
+use Friendica\Model\Contact;
+use Friendica\Protocol\ActivityPub\Transmitter;
+
+/**
+ * tool to access the system config from the CLI
+ *
+ * With this script you can access the system configuration of your node from
+ * the CLI. You can do both, reading current values stored in the database and
+ * set new values to config variables.
+ *
+ * Usage:
+ *   If you specify no parameters at the CLI, the script will list all config
+ *   variables defined.
+ *
+ *   If you specify one parameter, the script will list all config variables
+ *   defined in this section of the configuration (e.g. "system").
+ *
+ *   If you specify two parameters, the script will show you the current value
+ *   of the named configuration setting. (e.g. "system loglevel")
+ *
+ *   If you specify three parameters, the named configuration setting will be
+ *   set to the value of the last parameter. (e.g. "system loglevel 0" will
+ *   disable logging)
+ */
+class Relay extends \Asika\SimpleConsole\Console
+{
+	protected $helpOptions = ['h', 'help', '?'];
+
+	/**
+	 * @var App\Mode
+	 */
+	private $appMode;
+
+	protected function getHelp()
+	{
+		$help = <<<HELP
+console relay - Manage ActivityPub relay configuration
+Synopsis
+	bin/console relay [-h|--help|-?] [-v]
+	bin/console relay add <actor> [-h|--help|-?] [-v]
+	bin/console relay remove <actoor> [-h|--help|-?] [-v]
+
+Description
+	bin/console relay
+		Lists all active relais
+
+	bin/console relay add <actor>
+		Add a relay actor
+
+	bin/console relay remove <actoor>
+		Remove a relay actor
+
+Options
+    -h|--help|-? Show help information
+    -v           Show more debug information.
+HELP;
+		return $help;
+	}
+
+	public function __construct(App\Mode $appMode, array $argv = null)
+	{
+		parent::__construct($argv);
+
+		$this->appMode = $appMode;
+	}
```

Comment:
If you don't need `AppMode` anywhere in this console command, so in theory you could remove this constructor entirely. However, it looks like you're using the database in the console command, so you should include it in the constructor parameters like this:
```suggestion
	/** @var $dba Friendica\Database\Database */
	private $dba;

	public function __construct(Friendica\Database\Database $dba, array $argv = null)
	{
		parent::__construct($argv);

		$this->dba = $dba;
	}
```

Reply:
Yeah, I did some copy and paste from some other command line command.